config/routes/find.rb

```diff
@@ -15,6 +15,8 @@
   get '/course/:provider_code/:course_code', to: 'courses#show', as: 'course'
   get '/course/:provider_code/:course_code/placements', to: 'placements#index', as: :placements
   get '/course/:provider_code/:course_code/apply', to: 'courses#apply', as: :apply
+  get '/course/:provider_code/:course_code/git/:git_path', to: 'courses#git_redirect', as: :get_into_teaching_redirect
+  get '/course/:provider_code/:course_code/provider_website', to: 'courses#provider_website', as: :provider_website
```

Comment:
```suggestion
  get '/course/:provider_code/:course_code/provider/website', to: 'courses#provider_website', as: :provider_website
```

Not a massive suggestion but thought this would leave the door open for adding other links related to the provider.